{
  "modified": "2025-08-28T07:10:49Z",
  "published": "2025-08-28T07:10:49Z",
  "schema_version": "1.5.0",
  "id": "",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "config4py",
        "purl": "pkg:pypi/config4py"
      },
      "versions": [
        "0.0.1",
        "0.0.2",
        "0.0.3",
        "0.0.4",
        "0.0.5",
        "0.0.16",
        "0.0.18",
        "0.0.20",
        "0.0.21"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "e92bcf7e72e0d7c6bcf8169ed1c2d41945ddd96778ee0a62ff6d9520b8807a2c",
        "import_time": "2025-08-29T06:41:40.325365748Z",
        "id": "RLMA-2025-04131",
        "modified_time": "2025-08-28T07:10:49Z",
        "versions": [
          "0.0.1",
          "0.0.2",
          "0.0.3",
          "0.0.4",
          "0.0.5",
          "0.0.16",
          "0.0.18",
          "0.0.20",
          "0.0.21"
        ]
      }
    ]
  }
}
